prac_06/car.py

```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=0, name=""):
```

Comment:
Maybe swap the fuel and name parameters here?
(Based on the order of how you are returning them on line 16)